cardano-sl.cabal

```diff
@@ -120,6 +120,7 @@ library
                         Pos.Util.Arbitrary
                         Pos.Util.BackupPhrase
                         Pos.Util.JsonLog
+                        Pos.Util.Servant
```

Comment:
Ah, rly.
Then everything here seems to work fine. All modules that depend on `Util.Servant` presence are built successfully with `--no-wallet`